refinery/tool_manager/models.py

```diff
@@ -428,7 +430,11 @@ def _create_container_input_dict(self):
         return {
             self.FILE_RELATIONSHIPS: self.get_file_relationships_urls(),
             ToolDefinition.PARAMETERS: self._get_visualization_parameters(),
-            self.NODE_INFORMATION: self._get_detailed_input_nodes_dict()
+            self.NODE_INFORMATION: self._get_detailed_input_nodes_dict(),
+            ToolDefinition.EXTRA_DIRECTORIES: (
+                self.tool_definition.get_extra_directories()
+            )
```

Comment:
I don't think this should block, but the mix of `self` and `ToolDefinition`, or public and underscored confuses me, a little. Are those distinctions useful, and meaningful?